tracker/tracker/src/main/app/nodes.ts

```diff
@@ -2,13 +2,21 @@ type NodeCallback = (node: Node, isStart: boolean) => void;
 type ElementListener = [string, EventListener];
 
 export default class Nodes {
-  private readonly nodes: Array<Node | undefined>;
+  private nodes: Array<Node | undefined>;
   private readonly nodeCallbacks: Array<NodeCallback>;
   private readonly elementListeners: Map<number, Array<ElementListener>>;
+  gc: NodeJS.Timer | undefined;
   constructor(private readonly node_id: string) {
     this.nodes = [];
     this.nodeCallbacks = [];
     this.elementListeners = new Map();
+
+    this.gc = setInterval(() => {
+      // @ts-ignore should use settings object here
+      if (window.performance.memory.usedJSHeapSize > 500*1000*1000) {
```

Comment:
I think firefox doesn't have this API -> performance.memory.

Reply:
Ref:  https://bugzilla.mozilla.org/show_bug.cgi?id=1124223

Reply:
after some research I've made a completely different approach